src/Symfony/Component/Validator/Tests/Constraints/WhenTest.php

```diff
@@ -95,6 +100,19 @@ public function testAttributes()
             new NotNull(groups: ['Default', 'WhenTestWithAttributes']),
             new NotBlank(groups: ['Default', 'WhenTestWithAttributes']),
         ], $bazConstraint->constraints);
+        self::assertEmpty($bazConstraint->otherwise);
         self::assertSame(['Default', 'WhenTestWithAttributes'], $bazConstraint->groups);
+
+        [$quuxConstraint] = $metadata->properties['quux']->getConstraints();
+
+        self::assertInstanceOf(When::class, $quuxConstraint);
+        self::assertSame('true', $quuxConstraint->expression);
+        self::assertEquals([
+            new NotNull(groups: ['foo']),
+        ], $quuxConstraint->constraints);
+        self::assertEquals([
+            new Length(exactly: 10, groups: ['foo']),
+        ], $quuxConstraint->otherwise);
```

Comment:
```suggestion
        self::assertEquals([new NotNull(groups: ['foo'])], $quuxConstraint->constraints);
        self::assertEquals([new Length(exactly: 10, groups: ['foo'])], $quuxConstraint->otherwise);
```